src/ClassNameLayerResolver.php

```diff
@@ -59,4 +60,11 @@ public function getLayersByClassName(string $className): array
 
         return array_keys($layers);
     }
+
+    public function getLayers(): array
```

Comment:
`getLayerNames()`? Is it actually required that we do the filtering here instead of letting the consumer decide which information of the layers they are interested in?

Reply:
Not sure if I understand your question, but you couldn't you define the layers you want to see in the yaml config file, right. So if I want to see different layers, I would create different config files.

Reply:
Should be renamed to `getLayerNames` as we return names of the layers. But instead just returning names I think we should return layer-objects.

Reply:
What about the `getLayersByClassName(...)`? It is returning names as well. I would rename it too, ok?

Reply:
Layers are represented as a string. I am fine keeping these names. But we should rename this class to e.g. `LayerResolver`.